kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/lookup/UaDisbursementPayeeLookupableHelperServiceImpl.java

```diff
@@ -36,17 +41,18 @@
 	/**
 	 * @see org.kuali.kfs.fp.businessobject.lookup.DisbursementPayeeLookupableHelperServiceImpl#getPersonAsPayees(java.util.Map)
 	 */
+	@SuppressWarnings("deprecation")
 	protected List<DisbursementPayee> getPersonAsPayees(Map<String, String> fieldValues) {
 		List<DisbursementPayee> payeeList = new ArrayList<DisbursementPayee>();
 
-		List<String> employeeStatusCodes = (List<String>) CoreFrameworkServiceLocator.getParameterService().getParameterValuesAsString(DisbursementVoucherDocument.class, UaKFSConstants.ACTIVE_EMPLOYEE_STATUS_CODES_PARAM_NAME);
+		ArrayList<String> employeeStatusCodes = new ArrayList<String>(CoreFrameworkServiceLocator.getParameterService().getParameterValuesAsString(DisbursementVoucherDocument.class, UaKFSConstants.ACTIVE_EMPLOYEE_STATUS_CODES_PARAM_NAME));

```

Comment:
I'm not sure why, but the original casting to a List was causing an exception, and when I attempted to just use Collection<String> Exceptions were occurring when iterating over it (possibly the iteration has to be done in a different way?), so I used this to convert to an ArrayList.